docs/advanced/customization-guide.md

```diff
@@ -47,6 +50,14 @@ spec:
     - name: "my sample rule"
       labels:
         "my-sample-feature": "true"
+# Using tainting feature requires the feature gate (--no-taint) to be enabled (set to false)
+      taints:
+        - effect: PreferNoSchedule
+          key: bar
+          value: true
```

Comment:
You need to put `true` inside quotes, otherwise the unmarshaller treats it as bool (instead of string which is what we want) and fails

```suggestion
          value: "true"
```